tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "inlineSources": false,
-    "module": "ES6",
+    "module": "ES2020",
     "moduleResolution": "node",
     "target": "ES2017"
```

Comment:
This can be `ES2019` which is what Node.js 12 supports.

Reply:
I would also strongly recommend enabling strict mode: https://github.com/sindresorhus/tsconfig/blob/640e4bfe1172e27d59bf4d73a2c5df517cc01e1d/tsconfig.json#L21-L28

Reply:
Strict mode enabled in:
* #545